examples/viewer.cpp

```diff
@@ -135,7 +150,7 @@ bool Viewer::render()
     // wipe the drawing surface clear
     glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 
-    GLint x = 0, y = 0, width = 600, height = 400;
```

Comment:
@xlz Here's the main culprit: hardcoding the frame width and height instead of using relative sizing based on the actual window width and height.